src/metabase/api/search.clj

```diff
@@ -283,6 +285,8 @@
   (-> (base-query-for-model model search-ctx)
       (sql.helpers/left-join [:report_card :model]
                              [:= :model.id :action.model_id])
+      (sql.helpers/left-join :query_action
+                             [:= :query_action.action_id :action.id])
```

Comment:
Join in the `QueryAction` so that we can use its columns in `columns-for-model` below.